src/Service.Tests/TestHelper.cs

```diff
@@ -114,7 +115,10 @@ public static RuntimeConfig AddMissingEntitiesToConfig(RuntimeConfig config, str
         public const string SAMPLE_SCHEMA_DATA_SOURCE = SCHEMA_PROPERTY + "," + @"
             ""data-source"": {
               ""database-type"": ""mssql"",
-              ""connection-string"": ""testconnectionstring""
+              ""connection-string"": ""testconnectionstring"",
+              ""options"": {
```

Comment:
CLI, explicitly writes out true/false depending on whether session-context is enabled or not. So, adding this